tsuru/resource_tsuru_router_test.go

```diff
@@ -0,0 +1,72 @@
+package tsuru
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	echo "github.com/labstack/echo/v4"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/tsuru/go-tsuruclient/pkg/tsuru"
+)
+
+func TestAccTsuruRouter_basic(t *testing.T) {
+	fakeServer := echo.New()
+	fakeServer.POST("/1.8/routers", func(c echo.Context) error {
+		p := &tsuru.Router{}
+		err := c.Bind(&p)
+		require.NoError(t, err)
+		assert.Equal(t, p.Name, "test_router")
```

Comment:
why are we only checking the name of the router here? shouldn't we check for the type and config received also?